### PR TITLE
MJR changes

### PR DESCRIPTION
**Pull request by southeo**
- DB changes to MasJobRecord table
- Adds "masInput" field to Mas table
- Environmental variables for backend

### CODE REVIEW

File: test/backend/backend-deployment.yaml [hunk 1]
```diff
@@ -74,6 +74,25 @@ spec:
               value: kafka-cluster-kafka-bootstrap.kafka.svc.cluster.local:9092
             - name: application.base-url
               value: https://dev.dissco.tech
+            - name: fdo.profile
+              value: https://hdl.handle.net/21.T11148/532ce6796e2828dd2be6
+            - name: fdo.agent
+              value: https://ror.org/0566bfb96
+            - name: fdo.digital-object-type
+              value: https://hdl.handle.net/21.T11148/532ce6796e2828dd2be6
```
Comment (samleeflang) [inline test/backend/backend-deployment.yaml:82]: This is a temporary handle? Is it going to change that the profile and type are the same?

File: database/dissco/V1__create_tables.sql [hunk 2]
```diff
@@ -29,16 +29,22 @@ create index annotation_id_target_id_index
...
 create table mas_job_record
 (
-    job_id         uuid default uuid_generate_v4() not null
+    job_id         text                     not null
         constraint mas_job_record_pk
             primary key,
-    state          text                            not null,
-    creator_id     text                            not null,
-    time_started   timestamp with time zone        not null,
+    job_state      text                     not null
```
Comment (samleeflang) [inline database/dissco/V1__create_tables.sql:35]: Not really happy with this as this isn't an enum type. This means that we don't really get the advantages. I also wonder if the check doesn't impact performance, probably not and not that relevant here but still.

File: database/dissco/V1__create_tables.sql [hunk 3]
```diff
@@ -49,18 +55,18 @@ create index mas_job_record_job_id_index
 
 create table digital_media_object
```
Comment (samleeflang) [inline database/dissco/V1__create_tables.sql:56]: These and the others are just formatting changes right?

**Review by samleeflang — COMMENTED** [review]
We should really use the postgres enum instead of the check. It should be possible with JOOQ, if you get stuck I might have an example if I dig in some old code.

File: database/dissco/V1__create_tables.sql (hunk 3 above)
Comment (southeo) [reply database/dissco/V1__create_tables.sql:54]: Yes

File: test/backend/backend-deployment.yaml (hunk 1 above)
Comment (southeo) [reply test/backend/backend-deployment.yaml:82]: Placeholders for now, we don't have a public pid profile/type for a "kernel object"

**Review by samleeflang — APPROVED** [review]

**Pull request merged**